Library/Formula/keybase.rb

```diff
@@ -1,33 +1,33 @@
 class Keybase < Formula
   desc "Command-line interface to Keybase.io"
   homepage "https://keybase.io/"
-  url "https://github.com/keybase/node-client/archive/v0.8.23.tar.gz"
-  sha256 "20e7e7e7a125ddb3b0e7023c3dcaf12f12c7b183822592b164ca11dd1e6f9a25"
-  head "https://github.com/keybase/node-client.git"
+  url "https://github.com/keybase/client/archive/v1.0.0-47.tar.gz"
+  sha256 "a026fa0df54909989a50d607c0ac942857aa4ca7de00d5adef8ef30f403a082d"
+
+  head "https://github.com/keybase/client.git"
+  version "1.0.0-47"
 
   bottle :unneeded
 
-  depends_on "node"
-  depends_on :gpg
+  depends_on "go" => :build
 
   def install
-    # remove self-update command
-    # https://github.com/keybase/keybase-issues/issues/1477
-    rm "lib/command/update.js"
-    inreplace "lib/command/all.js", '"update", ', ""
-    inreplace "lib/req.js", "keybase-installer", "brew update && brew upgrade keybase"
-
-    libexec.install Dir["*"]
-    (bin/"keybase").write <<-EOS.undent
-      #!/bin/sh
-      export KEYBASE_BIN="#{bin}/keybase"
-      exec "#{Formula["node"].opt_bin}/node" "#{libexec}/bin/main.js" "$@"
-    EOS
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = buildpath
+    ENV["GO15VENDOREXPERIMENT"] = "1"
+    mkdir_p "src/github.com/keybase/client/"
+    mv "go", "src/github.com/keybase/client/"
+
+    system "go", "build", "-a", "-tags", "production brew", "github.com/keybase/client/go/keybase"
+
+    bin.install "keybase"
+  end
+
+  def post_install
+    system "#{opt_bin}/keybase", "launchd", "restart", "homebrew.mxcl.keybase"
   end
```

Comment:
No, it works if no plist file is there.  

It's just to make sure that if they were running version 1.0.0 of the service that after `brew upgrade` to 1.0.1 it will restart the service.